src/app/_components/CarouselTemplate.tsx

```diff
@@ -1,13 +1,29 @@
 'use client';
+import Image from 'next/image';
+import Link from 'next/link';
 import { useState } from 'react';
-import Carousel from '@/components/Carousel/Carousel';
+import CarouselContainer from '@/components/Carousel/CarouselContainer';
+import ClassCard from '@/components/ClassPreview/ClassPreview';
+import { searchBestInstructorData } from '@/types/instructor';
 
-interface ICarouselTemplateProps {
-  mode: 'class' | 'instructor';
-  children: React.ReactNode;
+interface classMode {
```

Comment:
```suggestion
interface IClassMode {
```
아래 바꾸면서 이부분이랑 `instructorMode` 컨벤션도 맞춰주시면...🙇‍♀️